zerver/webhooks/slack/view.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from django.utils.translation import ugettext as _
+from django.http import HttpRequest, HttpResponse
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message, create_stream_if_needed
+from zerver.lib.response import json_success, json_error
+from zerver.lib.validator import check_string, check_int
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.models import UserProfile, Client
+
+ZULIP_MESSAGE_TEMPLATE = "**{message_sender}** has sent the following message:\n `{text}`"
+VALID_OPTIONS = ['0', '1']
```

Comment:
I'd create two variables here like:
SHOULD_BE MAPPED = '1'
SHOULD_NOT_BE_MAPPED = '0'


It'd be more readable to make if as below:
```
if channels_mapt_to_topics == SHOULD_BE_MAPPED:
```